gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -277,6 +277,13 @@
   public static final String DATA_PUBLISHER_PREFIX = "data.publisher";
 
   /**
+   * Metadata configuration properties used internally
+   */
+  public static final String DATA_PUBLISHER_METADATA_OUTPUT_DIR =  "metadata.output.dir";
```

Comment:
Please prefix both properties with "data.publisher." 
The second property should be something along the lines of "data.publisher.metadata.string" .